steps/compliance/dwp-issuing-office-esa/DWPIssuingOfficeEsa.js

```diff
@@ -18,6 +20,17 @@ class DWPIssuingOfficeEsa extends SaveToDraftStore {
   }
 
   get options() {
+    const useDLA = [benefitTypes.disabilityLivingAllowance];
+    const benefitType = get(this, 'journey.req.session.BenefitType.benefitType');
+    const isDla = useDLA.indexOf(benefitType) !== -1;
+
+    if (isDla) {
+      return DWPIssuingOfficeEsa.selectify([
```

Comment:
The name of this Class and fields within talk about ESA but we've now got DLA logic in here, is there anything that can be done about this or is it complicated?

Reply:
I think you can introduce a new step for DLA and withing MRNDate class next(), you can goTo the new class for DLA. Like -> goTo(this.journey.steps.DWPIssuingOfficeDla).if(isDLABenefit)

Prety much copieng everything in isDLA condintion into the new step.

Reply:
ok, I will try that